ground/src/main/java/com/google/android/ground/ui/common/MapConfig.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.ground.ui.common
+
+/** Configuration to apply on the rendered base map. */
+data class MapConfig(val tileOverlay: Boolean)
```

Comment:
Something like `showTileOverlays` is probably more self-describing.

Reply:
Done